api/Element.json

```
@@ -6581,7 +6581,8 @@
             ],
             "chrome_android": [
               {
-                "version_added": "37"
+                "version_added": "37",
+                "notes": "From version 92 returns a promise instead of <code>undefined</code>. The behavior reflects this proposed specification change: https://github.com/w3c/pointerlock/pull/49."
```

Ah, got it. There's a style suggestion here (adding a conventional comma) plus making a rich link instead of a bare URL.

This needs to be applied to the other notes in this PR, plus, presumably, Chrome desktop above (which seems to have been missed in the previous PR).

```suggestion
                "notes": "From version 92, returns a promise instead of <code>undefined</code>. The behavior reflects <a href='https://github.com/w3c/pointerlock/pull/49'>a proposed specification change</a>."
```